src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -985,6 +986,14 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode, callable $e
                             ->end()
                             ->defaultValue([])
                         ->end()
+                        ->arrayNode('globals')
+                            ->info('Global parameters.')
+                            ->example(['app_version' => 3.14])
+                            ->defaultValue([])
```

Comment:
this is useless as it is already the default behavior for prototypes array nodes.

Reply:
Good catch